Apps/Medication/src/Controllers/MedicationController.cs

```diff
@@ -81,7 +81,8 @@ public MedicationController(IMedicationService medicationService, IHttpContextAc
         [Authorize]
         public async Task<List<MedicationStatement>> GetMedications(string hdid)
         {
-            string phn = await this.patientService.GetPatientPHNAsync(hdid).ConfigureAwait(true);
+            string jwtString = this.Request.Headers["Authorization"][0];
```

Comment:
can you use this.Request.Header.Authorization instead of this? would rather use the strongly typed property for it but not sure if its accessible

Reply:
The problem with that one is that it requires a Class that takes two parameters, Bearer and Token. If we do that we have to split the incoming authorization token before being passed and re build with the same parameters here. IMO unnecessary